fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/JDBCQueryBuilder.java

```diff
@@ -248,12 +248,22 @@ protected JDBCOperator getOperator(Parameter queryParm) {
         JDBCOperator operator = null;
         Modifier modifier = queryParm.getModifier();
 
-        if (modifier != null) {
+        // In the case where a URI, we need specific behavior/manipulation
+        // so that URI defaults to EQ, unless... BELOW
+        if (Type.URI.compareTo(queryParm.getType()) == 0) {
+            if(modifier != null && Modifier.BELOW.compareTo(modifier) == 0) {
```

Comment:
seem we can just remove modifier != null.

Reply:
I'm sorry we cannot... when I did it throws a NPE